src/elastic-search/elastic-search.service.ts

```diff
@@ -161,10 +162,13 @@ export class ElasticSearchService implements OnModuleInit {
       await this.esService.indices.open({
         index,
       });
-      Logger.log(`Elasticsearch Index Created-> Index: ${index}`);
+      Logger.log(
+        `Elasticsearch Index Created-> Index: ${index}`,
+        "Elasticsearch",
+      );
       return HttpStatus.CREATED;
     } catch (error) {
-      Logger.error("createIndex failed-> ElasticSearchService", error);
+      console.error("createIndex failed-> ElasticSearchService", error);
```

Comment:
Why not using Logger here too?

Reply:
This is for our use case specific change, if you see the catch block it contains log errors and throw errors. Since